legacy/src/app/partials/nodelist/add-machine.html

```diff
@@ -90,18 +90,17 @@ <h3 class="page-header__dropdown-title">Add machine</h3>
             </div>
           </div>
           <div class="p-form__group row" data-ng-repeat="mac in machine.macs">
-            <label for="mac-address" class="p-form__label col-2" ng-class="{ 'is-required': mac === machine.macs[0] }">
-              <span data-ng-if="mac === machine.macs[0]">MAC Address</span>&nbsp;
+            <label for="mac-address-{$ $index $}" class="p-form__label col-2" ng-class="{ 'is-required': mac === machine.macs[0] }">
+              <span data-ng-if="mac === machine.macs[0]">MAC Address</span>
```

Comment:
perhaps I'm misunderstanding this, but won't this make an empty <label> for the second and subsequent macs?

also instead of checking for === the first machine.macs can't we just use `$first`? (re: https://docs.angularjs.org/api/ng/directive/ngRepeat )

Reply:
I agree I don't think we should have this conditional in the label content. It looks like it just determines if the field should be required or not.

Reply:
> perhaps I'm misunderstanding this, but won't this make an empty <label> for the second and subsequent macs?

That's exactly what it does!

![Screenshot_2020-04-09 Machines bolla MAAS](https://user-images.githubusercontent.com/25733845/78844858-5d515000-7a4a-11ea-8471-7d5e29fc138a.png)

> also instead of checking for === the first machine.macs can't we just use $first? (re: https://docs.angularjs.org/api/ng/directive/ngRepeat )

That's definitely the smarter way to do this. Generally with the angular pages I try to only change the things that are actually causing bugs (e.g. the input validation, not the labels), because often fixing/improving the current code leads to completely unrelated things breaking. This seems pretty innocuous though.